Changelog.md

```diff
@@ -8,6 +8,41 @@
 
 ### Ignition Physics 2.x.x (20XX-XX-XX)
 
+### Ignition Physics 2.2.0 (2020-07-30)
+
+1. Add Base and EntityManagement to tpeplugin
```

Comment:
The order of these entries might be reversed. Don't we usually put the latest changes at the top?

Reply:
You mean the order of the PRs? I think we don't usually follow any order. In fact, we'd often try to put them in random places to avoid conflicts when many PRs are adding entries at the same time :sweat_smile:

Reply:
Oh, okay. I'm all for avoiding merge conflicts.